src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Hl7v2/ADT_A01/ADT01-23-expected.json

```diff
@@ -173,6 +176,25 @@
         "url": "Encounter/14693cea-02b9-3743-9010-fa549107ea68"
       }
     },
+    {
+      "fullUrl": "urn:uuid:5f49ca69-cbda-3b30-8480-fa750dfee4a2",
+      "resource": {
+        "resourceType": "Location",
+        "id": "5f49ca69-cbda-3b30-8480-fa750dfee4a2",
+        "mode": "instance",
+        "physicalType": {
+          "coding": [
+            {
+              "system": "http://terminology.hl7.org/CodeSystem/location-physical-type"
```

Comment:
A system without code or display text does not make sense as it carries no information. We should consider removing the system if code or display text is not present.

Reply:
This is caused by the [PL](https://github.com/microsoft/FHIR-Converter/blob/dotliquid/data/Templates/Hl7v2/DataType/_PL.liquid) template, which only includes a `system` field. @ACMoretxj and I will look into this together.

Reply:
Yes, and actually many of our templates have the same issue. I'll write this into our template improving plan.